doc/nrf/known_issues.rst

```diff
@@ -1209,7 +1222,7 @@ NCSDK-16060: :ref:`peripheral_lbs` sample build fails when the :kconfig:option:`
 
   **Workaround:** Manually cherry-pick and apply commit with fix from ``main`` (commit hash: ``32c827b20f3c5ab85a359e572d366da310fe2767``).
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

Comment:
@MarekPieta isn't this fixed for 2.1.0 with https://github.com/nrfconnect/sdk-nrf/commit/b834ff8860f3a30fe19c99dbf4c0c99b0b017245?

Reply:
Yes, it should be fixed already

Reply:
Updated.

Reply:
Thanks, @greg-fer